vegbilder-client/src/recoil/selectors.ts

```diff
@@ -1,7 +1,9 @@
+import { getAvailableStatisticsFromOGC } from 'apis/VegbilderOGC/getAvailableStatisticsFromOGC';
 import { getAvailableYearsFromOGC } from 'apis/VegbilderOGC/getAvailableYearsFromOGC';
 import { debounce, groupBy } from 'lodash';
```

Comment:
Oj, her såg eg tilfeldigvis at det allereie er brukt ein `groupBy`-funksjon frå `lodash`. Kan denne brukast i staden for den som vi la til? Eg vil anta at den gjer det same.

Reply:
Ja, oppdaget den i ettertid jeg også, men glemte å bytte. Har gjort det nå 👍